### ISSUE TITLE
Allow an embedded video from the instance on the "About this instance" page

### ISSUE THREAD

**Issue by TILvids** [Chocobozzz/PeerTube#3154]
I have a video I've created about my instance for TILvids. I'd love to feature it on the "About this instance" page, but there's no real good way to do that, so I just had to do a standard link. It'd be nice to embed it and feature it somehow.

[Here's a link to the video, for context.](https://tilvids.com/videos/watch/69da6d3a-b56b-4b22-800d-8eb1530f4394)

**Comment by test2a** [comment]
did you do the voice over yourself? this is a super nice video and i wish you all the best in your endeavors. I checked the about instance on your website and i did not find any details about the hardware. Can you give out the actual stats like this instance does https://devtube.dev-wiki.de/about/instance . it would be super informative to understand the bandwidth requirements and stuff like CPU usage and other things. not just the live data, that would be even better but just having a simple , 2vcpu, 1gbps unmetered, 4gb ram and similar would be sufficient :-)

**Comment by TILvids** [comment]
Hi @test2a !

Thanks for the kind words about the video, and the encouragement! (yes, I did all of the video myself, I love making videos!) I really want to build a nice alternative video community, and PeerTube is making it about as easy as possible, at least on the technical side!

For the server, I'm having to constantly upgrade it, so it wasn't really worth putting on. At the moment, it's hosted on a VPS that has a single core with 2GB of memory. I believe the connection is only 100Mbps at the moment. Storage has gone from 20, to 40, and now 80GB (fortunately we have a few Patron donors now, so I'm going to be able to begin scaling more now). I'll have to upgrade storage first to 160GB. I'll likely next do an upgrade to 2 cores, 4GB memory, and 250Mbps. The 2 cores will be a big addition, because right now with more and more people starting to watch, the server is having to queue transcoding longer and longer (I think; I'm not 100% sure what PeerTube is doing behind the scenes, just something I've noticed). So having more cores will certainly help with that.

Happy to answer any other questions, thanks again for the support!
